guide/ch_advanced_use.tex

```diff
@@ -453,6 +453,38 @@ \subsubsection{NMEA Device}
 hardware and other settings, e.g. real GPS sensor or network and WiFi
 detection.\footnote{\url{https://support.microsoft.com/en-us/windows/windows-location-service-and-privacy-3a8eee0a-5b0b-dc07-eede-2a5ca1c49088}}
 
+
+\subsection{Modernized MESA3D libraries (Software OpenGL on Windows)}
+\label{sec:ExtraData:Mesa}
+
+Stellarium uses hardware-accelerated OpenGL for rendering. On very old Windows systems without dedicated hardware, 
+or in special circumstances like virtual machines, hardware accelerated graphics may not be available, 
+and the \program{Mesa3D} software implementation of OpenGL is used. This can also be forced by the command-line option \command{-\/-mesa-mode}. 
+Rendering the graphics without hardware acceleration is of course much slower than accelerated graphics, but it's surely better than nothing.
+
+Qt's default Mesa library (Mesa 11) provides support for OpenGL~3.0 only. This prevents users of Qt5-based versions from applying dithering. 
+On Qt6, also the \program{ShowMySky} skylight model (see section~\ref{sec:skylight:ShowMySky}) is then not available. 
+
+We have replaced this by a version of Mesa~20 compiled by Federico Dossena\footnote{\url{https://fdossena.com/?p=mesa/index.frag}} which appears to be even a bit faster.
+
+The \program{Mesa3D} project \footnote{\url{https://www.mesa3d.org/}} meanwhile (2023) provides at least OpenGL~4.5. 
+If you wish to use more ``up to date'' precompiled binaries and have Administrator privileges on your Windows system, \newFeature{23.3}
+you can replace this library using the following steps to install inofficial builds\footnote{\url{https://github.com/pal1000/mesa-dist-win/}}. 
+Note however that after this upgrade, the program runs with considerably ($\approx$35\%?) lower framerate. 
```

Comment:
Didn't you say this improves frame rate rather than degrades?

Reply:
This branch should
1. replace Qt's Mesa 11 by fdossena's Mesa 20. Unconditionally.
2. For "purists" who don't like taking Mesa DLLs from other sources than github, describe a path to move to a "more official" Mesa build.

(2) is done. The next steps will hopefully bring the Mesa20 DLL to where it belongs.

Reply:
(2) makes zero sense, because pal1000's builds are not more official than fdossena's ones. Both provide instructions to reproduce the builds, so you can makes your own binaries if you want. And GitHub is not an authority for Mesa in any way, since Mesa has long moved to GitLab@FreeDekstop.